### ISSUE TITLE
Document the need to manually include custom handlers when running a `@WebMvcTest`

### ISSUE THREAD

**Issue by donalmurtagh** [wimdeblauwe/error-handling-spring-boot-starter#106]
According to the docs

> There is nothing special to configure for unit tests that use @WebMvcTest. The library is automatically active when it is on the classpath.

However, i noticed that my custom handler bean is not included when running such a test, so it would be helpful to mention this in the docs. In my case, I added it to the test configuration like so

```java
@WebMvcTest(MyController.class)
@Import(MyCusomExceptionHandler.class)
class MyControllerTests {

}
```

**Comment by wimdeblauwe** [comment]
Yes, only the built-in handlers are loaded automatically indeed.
